telemetry/definitions/commonDefinitions.json

```diff
@@ -1598,6 +1598,19 @@
                 }
             ]
         },
+        {
+            "name": "amazonq_modifySourceFolder",
+            "description": "User modified source folder",
```

Comment:
> This could happen on different steps of our process, whenever we upload the repo to our backend (that happens at least once before planning the execution, and once more for each code generation requested).
> It is kinda like a sub-command of `/dev` but it’s not triggered by the user whenever they want, and only when we think the repo is too large and a sub-folder should be selected.
> We display the button and want to capture the action of the user clicking it.